examples/vg-specs/area_vertical.vg.json

```diff
@@ -3,6 +3,13 @@
     "description": "Area chart showing weight of cars over time (vertical).",
     "autosize": "pad",
     "padding": 5,
+    "encode": {
+        "update": {
+            "fill": {
+                "value": "transparent"
```

Comment:
I guess later we might only add this if there is a selection in the spec. Thanks for the clarification.